tests/wpt/mozilla/meta/css/meta_viewport_resize.html.ini

```diff
@@ -0,0 +1,2 @@
+prefs: ["layout.viewport.enabled:true"]
```

Comment:
I'm not sure if this format will work, since I'm used to seeing:
```
[xmlhttprequest-timeout-abortedonmain.html]
  type: testharness
```
before any other directives.